peps/pep-0363.rst

```diff
@@ -1,14 +1,12 @@
 PEP: 363
 Title: Syntax For Dynamic Attribute Access
-Version: $Revision$
-Last-Modified: $Date$
 Author: Ben North <ben@redfrontdoor.org>
 Status: Rejected
 Type: Standards Track
-Content-Type: text/x-rst
 Created: 29-Jan-2007
 Post-History: 12-Feb-2007
 
+.. rejected::
```

Comment:
Likewise, This PEP contains a part near the end discussion the rejection, its just rather long and buried within another section. If you added a ref target label below (e.g. `363-rejection`), you could link to the `Mailing Lists Discussion` section below describing the rejection reasons, e.g.

```rst
.. rejected::
   This PEP has been rejected.
   See :ref:`363-rejection` for more information.
```